### DISCUSSION TITLE
How to make editable connections?

### DISCUSSION THREAD

**Discussion by seepine** [bcakmakoglu/vue-flow#273] [Q&A] [answered]
Through doc, I can add nodes and establish connections, but I don't know how to delete connections

**Comment by seepine** [comment, marked as answer]
Maybe I should add the delete icon or implement the Del shortcut, but I don't know where to start

**Reply by bcakmakoglu** [reply]
"Connections" are called "Edges" in this context and to delete edges, you simply remove them from your elements array or use the helper function 
```ts
const { removeEdges } = useVueFlow()

removeEdges(['e1-2'])
```

**Reply by seepine** [reply]
i know removeEdges can remove edge,
but i dont know how to trigger reasonably

For example, I should add a delete icon to the line segment?

**Reply by bcakmakoglu** [reply]
Well, I can't tell you how to build your business logic 😅 
It really depends on how you want to delete it - if you want the edge to be deletable with a button press, do that.

By default, edges can be deleted by just selecting it (click on the edge, it will be marked as selected) and then just use the Backspace key (this is the default delete key handler, you can customize the key to whatever key you want).
Same goes for nodes.

If you need something else, you can implement that logic using the snippet above (using `removeEdges`).

**Reply by seepine** [reply]
thanks

**Reply by bcakmakoglu** [reply]
fyi: there is an example of an edge with a button that removes the edge [here](https://vueflow.dev/examples/edges/).
